### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tomasmik**
Updated readme

### CODE REVIEW

**Review by cube2222 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,37 +2,23 @@
...
-### Homebrew
-
-You can install `spacectl` using Homebrew on MacOS or Linux:
-
-```bash
-brew install spacelift-io/spacelift/spacectl
-```
-
-### Arch linux
+It's main purpose is to help you explore and execute actions inside Spacelift. It provides limited functionality for creating or editting resources. To do that programatically see the [Spacelift Terraform Provider](https://github.com/spacelift-io/terraform-provider-spacelift).
```
Comment (adamconnelly) [inline README.md:5]:
```suggestion
It's main purpose is to help you explore and execute actions inside Spacelift. It provides limited functionality for creating or editing resources. To do that programatically see the [Spacelift Terraform Provider](https://github.com/spacelift-io/terraform-provider-spacelift).
```

**Review by mbialon — APPROVED** [review]

**Review by b4k3r — APPROVED** [review]

**Review by adamconnelly — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (michieldewilde) [inline README.md:5]:
```suggestion
Its primary purpose is to help you explore and execute actions inside Spacelift. It provides limited functionality for creating or editing resources. To do that programmatically, you can use the [Spacelift Terraform Provider](https://github.com/spacelift-io/terraform-provider-spacelift).
```

File: README.md [hunk 2]
```diff
@@ -2,37 +2,23 @@
...
-Install [`spacectl-bin`](https://aur.archlinux.org/packages/spacectl-bin) from the Arch User Repository ([AUR](https://aur.archlinux.org/)):
+## Installation
 
-```bash
-yay -S spacectl-bin
-```
+### Officially supported packages
 
-Disclaimer: The package is community-maintained, please make sure to verify the [`PKGBUILD`](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=spacectl-bin) before installing/updating.
+Officially supported packages are maintained by [Spacelift](https://spacelift.io/) and are the best way to install `spacectl`
```
Comment (michieldewilde) [inline README.md:11]:
```suggestion
Officially supported packages are maintained by [Spacelift](https://spacelift.io/) and are the preferred ways to install `spacectl`
```

File: README.md [hunk 3]
```diff
@@ -242,6 +260,12 @@ Enter Spacelift endpoint (eg. https://unicorn.app.spacelift.io/):
 
 In the next step, you will be asked to choose which authentication method you are going to use. Note that if your account is using [SAML-based SSO authentication](https://docs.spacelift.io/integrations/single-sign-on), then API keys and login with a browser are your only options. After you're done entering credentials, the CLI will validate them against the server, and assuming that they're valid, will persist them in a credentials file in `.spacelift/${MY_ALIAS}`. It will also create a symlink in `${HOME}/.spacelift/current` pointing to the current profile.
 
+By default the login process is interactive, however if that not does fit your workflow the steps can be predefined using flags, for example:
```
Comment (michieldewilde) [inline README.md:263]:
```suggestion
By default, the login process is interactive, however, if that does not fit your workflow, the steps can be predefined using flags, for example:
```

**Review by michieldewilde — APPROVED** [review]
Some grammar and spelling errors.

**Comment by tomasmik** [comment]
Never seen so many approvals so fast. Thanks everyone!

**Pull request merged**